example/android/javalib/3-linting/1-basic-linting/build.mill

```diff
@@ -0,0 +1,56 @@
+package build
+
+import mill._
+import mill.javalib.android.{AndroidSdkModule, AndroidAppModule, AndroidLintModule}
+
+// Create and configure an Android SDK module to manage Android SDK paths and tools.
+object androidSdkModule0 extends AndroidSdkModule {
+  def buildToolsVersion = "35.0.0"
+  def bundleToolVersion = "1.17.2"
+}
+
+// Actual android application
+object app extends AndroidLintModule {
+  def androidSdkModule = mill.define.ModuleRef(androidSdkModule0)
+
+  // Set path to the location of the project source codes
+  def androidLintSources = Task { Some(PathRef(millSourcePath / "src" / "main" / "java")) }
+  
+  // Set path to the custom `lint.xml` config file. It is usually at the root of the project
+  def androidLintConfigPath = Task { Some(PathRef(millSourcePath / "lint.xml")) }
+
+  // Set path to generated `.jar` files in this case (or `.class` files)
+  def androidLintClasspath = Task { Some(androidJar()) }
+}
+
+/** See Also: app/AndroidManifest.xml */
+/** See Also: app/resources/values/colors.xml */
+/** See Also: app/resources/values/strings.xml */
+/** See Also: app/src/main/java/com/helloworld/app/MainActivity.java */
+/** See Also: app/lint.xml */
+
+// The `lint.xml` file shown above ignores the `MissingApplicationIcon` error in the code as
+// the project is for demostration. There are no icons for the demo project.
+
+
+/** Usage
+
+> ./mill show app.androidLintRun  # Display full path to the linting report in HTML
+".../out/app/androidLintRun.dest/report.html"
+
+> cat out/app/androidLintRun.dest/report.html" # Display content of the linting report
+Avoid hardcoding the debug mode; leaving it out allows debug and release builds to automatically assign one
```

Comment:
You probably need to use `...` wildcards here, unless the HTML file really only contains this string and nothing else. See https://mill-build.org/mill/extending/writing-plugins.html#_example_tests